### PR TITLE
Des 3542 - Support user authentication - WPF

### PR DESCRIPTION
**Pull request by matthewgardner**

### CODE REVIEW

File: minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs [hunk 1]
```diff
@@ -47,5 +73,30 @@ protected override bool OnCertificateError(CefBrowser browser, CefErrorCode cert
             Logger.Error("Failed to load resource due to an invalid certificate: " + requestUrl);
             return base.OnCertificateError(browser, certError, requestUrl, sslInfo, callback);
         }
+        protected override bool GetAuthCredentials(CefBrowser browser, CefFrame frame, bool isProxy, string host, int port, string realm, string scheme, CefAuthCallback callback)
+        {
+            string strFriendlyName = AppDomain.CurrentDomain.FriendlyName;
+            Process[] pro = Process.GetProcessesByName(strFriendlyName.Substring(0, strFriendlyName.LastIndexOf('.')));
+            System.Windows.Forms.IWin32Window handle = new WindowWrapper(pro[0].MainWindowHandle);
```
Comment (lneir) [inline minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs]: are lines 78 to 80 still needed???

File: minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs [hunk 2]
```diff
@@ -1,9 +1,35 @@
...
+using Paragon.Runtime.WPF;
+using System.Windows.Forms;
+using System.Diagnostics;
+using System.Windows;
+using System.Windows.Interop;
+
 
 namespace Paragon.Runtime
 {
+    public class WindowWrapper : System.Windows.Forms.IWin32Window
```
Comment (lneir) [inline minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs]: is WindowWrapper class still needed?

File: minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs (hunk 1 above)
Comment (matthewgardner) [inline minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs]: ek - how did that get in there?! (sorry I am still learning how to use git :*) )

Comment (lneir) [inline minuet/symphony/Paragon.Runtime/CefHandlers/CefWebRequestHandler.cs]: np, just remove and re-submit pr.

**Pull request closed without merging**